### PR TITLE
feat(py_wheel): Add support for specifying Project-URL in METADATA

### PR DESCRIPTION
**Pull request by sfc-gh-wzhao**
`Project-URL` is a field available in core metadata since version 1.2, which allows specifying additional URLs and display as Project Links in PyPI package web page.

https://packaging.python.org/en/latest/specifications/core-metadata/#project-url-multiple-use

This change adds the support to specify that.

### CODE REVIEW

**Comment by sfc-gh-wzhao** [comment]
@sfc-gh-sdas FYI.

File: python/private/py_wheel.bzl [hunk 1]
```diff
@@ -301,6 +305,9 @@ def _py_wheel_impl(ctx):
     if ctx.attr.summary:
         metadata_contents.append("Summary: %s" % ctx.attr.summary)
 
+    for label, url in sorted(ctx.attr.project_urls.items()):
```
Comment (chrislovecnm) [inline python/private/py_wheel.bzl:308]: Please check for the max length of the label. The label is limited to 32 characters.

File: python/private/py_wheel.bzl [hunk 2]
```diff
@@ -176,6 +176,10 @@ _other_attrs = {
         doc = "A string specifying the license of the package.",
         default = "",
     ),
+    "project_urls": attr.string_dict(
+        doc = ("A string dict specifying additional browsable URLs for the project and corresponding labels. " +
```
Comment (chrislovecnm) [inline python/private/py_wheel.bzl:180]: Nit pick. Mention the key first and the label second. You have them reversed.

**Review by chrislovecnm — COMMENTED** [review]
Thanks for the PR! See comments.

File: python/private/py_wheel.bzl (hunk 1 above)
Comment (sfc-gh-wzhao) [reply python/private/py_wheel.bzl:310]: Thank you for pointing out. A check has been added, and will fail out if the length is over limit. Corresponding test is also added.

File: python/private/py_wheel.bzl (hunk 2 above)
Comment (sfc-gh-wzhao) [reply python/private/py_wheel.bzl]: Updated with better explanation about the way to specify.

**Review by chrislovecnm — APPROVED** [review]
LGTM!

**Pull request merged**